.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
```

Comment:
/.idea vergeten toe te voegen. Dit kan je altijd checken door git status te gebruiken voor je git commit gegruikt. Als dat je idea map er in staat kan je deze verwijderen met git rm --cached -r .idea